### PR TITLE
[CARBONDATA-3743] Added pre-priming check in the Spark job description page

### PR DESCRIPTION
**Pull request by vikramahuja1001**
### Why is this PR needed?
 Added pre-priming check in the Spark job description page
 
 ### What changes were proposed in this PR?

    
 ### Does this PR introduce any user interface change?
 - No
 - Yes. (please explain the change and update document)

 ### Is any new testcase added?
 - No
 - Yes

### CODE REVIEW

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/2456/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.4/748/

**Comment by Indhumathi27** [comment]
@vikramahuja1001 Please update the PR description, Changes Added

File: integration/spark/src/main/scala/org/apache/carbondata/indexserver/IndexServer.scala [hunk 1]
```diff
@@ -131,8 +131,15 @@ object IndexServer extends ServerInterface {
       lazy val getCountTask = {
         if (!request.isFallbackJob) {
           sparkSession.sparkContext.setLocalProperty("spark.jobGroup.id", request.getTaskGroupId)
-          sparkSession.sparkContext
-            .setLocalProperty("spark.job.description", request.getTaskGroupDesc)
+          if (!request.ifAsyncCall()) {
```
Comment (Indhumathi27) [inline integration/spark/src/main/scala/org/apache/carbondata/indexserver/IndexServer.scala]: Can simplify the code like below:
          val taskGroupDesc = if (!request.ifAsyncCall()) {
            request.getTaskGroupDesc
          } else {
            "PrePriming Job Fired for table: " + request.getCarbonTable.getDatabaseName +
            CarbonCommonConstants.POINT + request.getCarbonTable.getTableName
          }
          sparkSession.sparkContext.setLocalProperty("spark.job.description", taskGroupDesc)

**Comment by kunal642** [comment]
@vikramahuja1001 Please fix comments

File: integration/spark/src/main/scala/org/apache/carbondata/indexserver/IndexServer.scala (hunk 1 above)
Comment (vikramahuja1001) [reply integration/spark/src/main/scala/org/apache/carbondata/indexserver/IndexServer.scala]: done

**Comment by CarbonDataQA1** [comment]
Build Failed  with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/932/

**Comment by CarbonDataQA1** [comment]
Build Failed  with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/2642/

**Comment by vikramahuja1001** [comment]
retest this please

**Comment by CarbonDataQA1** [comment]
Build Failed  with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/937/

**Comment by CarbonDataQA1** [comment]
Build Failed  with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/2647/

**Comment by kunal642** [comment]
@vikramahuja1001 Please fix the build

**Comment by vikramahuja1001** [comment]
@kunal642 , the build is completing. There seems to be some issue with jenkins.

**Comment by kunal642** [comment]
retest this please

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.4.5, Please check CI http://121.244.95.60:12545/job/ApacheCarbon_PR_Builder_2.4.5/967/

**Comment by CarbonDataQA1** [comment]
Build Success with Spark 2.3.4, Please check CI http://121.244.95.60:12545/job/ApacheCarbonPRBuilder2.3/2680/

**Comment by kunal642** [comment]
LGTM

**Pull request closed without merging**